core/src/main/java/org/apache/carbondata/core/util/DeleteLoadFolders.java

```diff
@@ -90,13 +93,13 @@ public static void physicalFactAndMeasureMetadataDeletion(CarbonTable carbonTabl
    * Delete the invalid data physically from table.
    * @param carbonTable table
    * @param loadDetails Load details which need clean up
-   * @param isForceDelete is Force delete requested by user
+   * @param force Force delete Compacted and MFD segments
```

Comment:
force --> isForceDelete
Also it will force clean the trash. add that also in header description